tests/test_classifier/test_rocauc.py

```diff
@@ -88,6 +88,60 @@ def test_binary_rocauc(self):
         visualizer.poof()
         self.assert_images_similar(visualizer)
 
+    def test_binary_probability_decision(self):
```

Comment:
Stellar test cases for the models that have various combinations of `decision_function` and `predict_proba`! Very nice work!

Reply:
Thanks!